notebooks/features/classification/Classification - Adult Census.ipynb

```diff
@@ -93,12 +90,17 @@
    "cell_type": "code",
    "execution_count": null,
    "source": [
-    "if os.environ.get(\"AZURE_SERVICE\", None) != \"Microsoft.ProjectArcadia\":\n",
-    "    model.write().overwrite().save(\"dbfs:/AdultCensus.mml\")\n",
-    "else:\n",
+    "platform = current_platform()\n",
+    "if platform == \"synapse\":\n",
```

Comment:
I know its a PITA but can we replace all of the places in the notebooks where we have this silly project arcadia thing?

Reply:
Yep - that was the plan -- wanted to see if this works fine and does not regress on E2E tests